src/App/Pages/Vault/ScanPage.xaml

```diff
@@ -34,16 +34,15 @@
             <RowDefinition Height="*" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <zxing:ZXingScannerView
-            x:Name="_zxing"
+        <ContentView 
+            x:Name="_scannerContainer"
             HorizontalOptions="FillAndExpand"
             VerticalOptions="FillAndExpand"
             AutomationId="zxingScannerView"
             IsVisible="{Binding ShowScanner}"
             Grid.Column="0"
             Grid.Row="0"
-            Grid.RowSpan="3"
-            OnScanResult="OnScanResult"/>
+            Grid.RowSpan="3"/>
```

Comment:
Oh ye, `LazyView` might do the trick! Let me try it out.

Reply:
After trying it out i don't think the `LazyView` will be very useful in this case. I would have to do a custom implementation with the initialization just for this one view. If we were to use the scanner in other places it might be useful but in this case i don't think so.